cmd/snap/cmd_run.go

```diff
@@ -954,3 +967,88 @@ func (x *cmdRun) runSnapConfine(info *snap.Info, securityTag, snapApp, hook stri
 		return syscallExec(cmd[0], cmd, env)
 	}
 }
+
+func randomUUID() (string, error) {
+	uuidBytes, err := ioutil.ReadFile("/proc/sys/kernel/random/uuid")
+	return strings.TrimSpace(string(uuidBytes)), err
+}
+
+var createTransientScope = func(securityTag string) error {
```

Comment:
is this function unit tested somehow? only spread tested?

Reply:
It's only spread tested at the moment. I tried doing an unit test for it but got stuck on dbus parts. I can iterate and get one. added for real though.